### PR TITLE
Adding Japanese README

### PR DESCRIPTION
**Pull request by beachside-project**
Adding Japanese README: README.ja-jp.md

### CODE REVIEW

File: README.ja-jp.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+Form Recognizer の改善にご協力ください。[Take our survey!](https://aka.ms/FR-HaTS-Survey)
+
+## Features Preview
+
+[![Build Status](https://dev.azure.com/msazure/Cognitive%20Services/_apis/build/status/microsoft.OCR-Form-Tools?branchName=master)](https://dev.azure.com/msazure/Cognitive%20Services/_build/latest?definitionId=118293&branchName=master)
+
+[Form Recognizer](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/) のオープンソースの Labeling Tool で、Form OCR Test Toolset (FOTT) の一部です。
+
+これはツールの `MAINP ブランチです。これは、利用可能なすべての最新機能を含んでいます。プレビューのため、最も安定したバージョンでは**ありません**。
```
Comment (cschenio) [inline README.ja-jp.md:12]: MAINP -> MAIN

**Review by cschenio — CHANGES_REQUESTED** [review]

File: README.ja-jp.md (hunk 1 above)
Comment (beachside-project) [reply README.ja-jp.md]: Thank you for your review! Fixed this.

File: README.ja-jp.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+FOTT の Labeling Tool は、TypeScript を使った React + Redux の Web アプリケーションです。このプロジェクトは [Create React App](https://github.com/facebook/create-react-app) でブートストラップしています。
+
+現在の Labeling Toolの機能 ([こちら](https://youtu.be/GRGs18lZJ1E?t=462)でショートデモが見れます)
+
+* PDF、JPEG、TIFF 形式のフォームに**ラベルを付ける**
+* [Form Recognizer](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/) でラベル付けされたデータを使ってモデルを**学習する**
+* 学習したモデルを使ってフォームを**予測／分析**をし、そのフォームから予測／分析した結果のキー・バリューを抽出
+
+## Getting Started
```
Comment (cschenio) [inline README.ja-jp.md:28]: suggestion: would you mind also translating the headings in this readme?

**Review by cschenio — CHANGES_REQUESTED** [review]

File: README.ja-jp.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+左ペインの "Train" アイコンをクリックして "Training" ページを開きます。次に "Train" ボタンをクリックしてモデルのトレーニングを開始します。トレーニングプロセスが完了すると、以下の情報が表示されます。
+
+* **Model ID** - 作成してトレーニングされたモデルの ID です。トレーニングを行うたびに独自の ID を持つ新しいモデルが作成されます。この文字列を安全な場所にコピーしてください。REST API を使って予測・分析を行う場合に必要となります。
+* **Average Accuracy** - モデルの平均精度です。追加のフォームにラベルを付け再度トレーニングを行ってモデルを作成することで、モデルの精度を向上させることができます。まずは5つのフォームにラベルを付け、必要に応じてフォームを追加することをお勧めします。
+* タグの一覧と、タグごとの推定精度が表示されます。
+
+![alt text](docs/images/train-model.png "Train Model")
+
+トレーニングが終了したら **Average Accuracy** の値を確認します。もし verage Accuracy  が低い場合は、ドキュメントを追加して上記の手順を繰り返す必要があります。ラベル付けされたドキュメントは、プロジェクトのインデックスに残ります。
```
Comment (cschenio) [inline README.ja-jp.md:180]: verage Accuracy -> Average Accuracy

File: README.ja-jp.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+[こちら](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/label-tool#set-up-the-sample-labeling-tool)の説明をご覧ください。また、最新のコンテナーイメージの情報については、[こちら](https://hub.docker.com/_/microsoft-azure-cognitive-services-custom-form-labeltool?tab=description)の docker hub リポジトリをご覧ください。
+
+* `latest` の docker image tags は、FOTT の一般利用可能なリリースをトラッキングしています。
+* `latest-preview` の docker image tags は、FOTT のプレビューリリースをトラッキングします。
+* `latest-preview-private` の docker image tags は、FOTT のプライベートプレビューリリースをトラッキングします。
+
+### Run as web application
+
+最新のWebブラウザを使って、FOTTを直接起動することができます:
```
Comment (cschenio) [inline README.ja-jp.md:52]: Would you mind adding space between 全角 and 半角 characters? (Just like this comment itself)

File: README.ja-jp.md (hunk 3 above)
Comment (cschenio) [inline README.ja-jp.md:180]: verage Accuracy -> Average Accuracy

**Review by cschenio — CHANGES_REQUESTED** [review]

File: README.ja-jp.md (hunk 2 above)
Comment (beachside-project) [reply README.ja-jp.md:28]: Thanks! I had unconsciously missed it. I added the translation.

File: README.ja-jp.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+
+タグのタイプとフォーマットは、タグのコンテキストメニューで指定できます。タイプとフォーマットの情報は、ソースと同じ場所の fields.json に保存されます。これらの情報は、より良い結果を得るために後処理で使用されます。
+
+![alt text](docs/images/specify-tag-type-format.png "Specify Tag Type and Format")
+
+#### カスタムモデルをトレーニングする
+
+左ペインの "Train" アイコンをクリックして "Training" ページを開きます。次に "Train" ボタンをクリックしてモデルのトレーニングを開始します。トレーニングプロセスが完了すると、以下の情報が表示されます。
+
+* **Model ID** - 作成してトレーニングされたモデルの ID です。トレーニングを行うたびに独自の ID を持つ新しいモデルが作成されます。この文字列を安全な場所にコピーしてください。REST API を使って予測・分析を行う場合に必要となります。
```
Comment (cschenio) [inline README.ja-jp.md:174]: suggestion: た

**Comment by cschenio** [comment]
Thank you very much @beachside-project! Your README contribution values a lot to our language expansion part and increases the accessibility of this open-source tool to the Japanese audience.

File: README.ja-jp.md (hunk 4 above)
Comment (beachside-project) [reply README.ja-jp.md]: Thanks! I fixed it.

File: README.ja-jp.md (hunk 3 above)
Comment (beachside-project) [reply README.ja-jp.md]: Thanks! I fixed it.

File: README.ja-jp.md (hunk 5 above)
Comment (beachside-project) [reply README.ja-jp.md:174]: Is there any suggestion?

**Review by buddhawang — APPROVED** [review]
LGTM, thanks @beachside-project!

**Pull request merged**